### PR TITLE
Fix EXPERIMENTAL_useTableMeasures SSR errors

### PR DESCRIPTION
**Pull request by thayannevls**
#### What is the purpose of this pull request?
Fixing this error:
Screenshot from 2020-03-13 11-24-12
 

#### What problem is this solving?
[Fixed workspace](https://thay2--cosmetics1.myvtex.com/wedemo)

[Broken workspace](https://thay--cosmetics1.myvtex.com/wedemo)

#### Types of changes

- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/EXPERIMENTAL_Table/hooks/useTableMeasures.tsx [hunk 1]
```diff
@@ -41,8 +41,10 @@ export function calculateTableHeight(
 }
 
 export function getScrollbarWidth(): number {
-  if (!window || !document || !document.documentElement)
+  const isSSR = typeof document === 'undefined'
```
Comment (emersonlaurentino) [inline react/components/EXPERIMENTAL_Table/hooks/useTableMeasures.tsx]: Shouldn't it be like that?
```suggestion
  const isSSR = typeof document === undefined
```

Comment (matheusps) [reply react/components/EXPERIMENTAL_Table/hooks/useTableMeasures.tsx]: nope, it compares as a string [typeof ref](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Operators/typeof)

Comment (thayannevls) [reply react/components/EXPERIMENTAL_Table/hooks/useTableMeasures.tsx]: The `typeof` operator always return a string with the type :smile:

Comment (lucasmedeiros) [inline react/components/EXPERIMENTAL_Table/hooks/useTableMeasures.tsx]: Shouldn't we check if `window` is also not set like that, or it isn't necessary?

```suggestion
  const isSSR = typeof document === 'undefined' || !window
```

Comment (thayannevls) [reply react/components/EXPERIMENTAL_Table/hooks/useTableMeasures.tsx]: Done, thanks :smile:

File: react/components/EXPERIMENTAL_Table/hooks/useTableMeasures.tsx [hunk 2]
```diff
@@ -39,8 +39,12 @@ export function calculateTableHeight(
 }
 
 export function getScrollbarWidth(): number {
-  if (!window || !document || !document.documentElement)
+  const isSSR =
+    typeof document === 'undefined' || typeof document === 'undefined'
```
Comment (vitoriaHeliane) [inline react/components/EXPERIMENTAL_Table/hooks/useTableMeasures.tsx]: Why are you using the same terms in the logic expression?

Comment (thayannevls) [reply react/components/EXPERIMENTAL_Table/hooks/useTableMeasures.tsx]: True :woman_facepalming: I'll launch again fixing this

**Pull request merged**